applications/tasks/src/modules/kt_rates.erl

```diff
@@ -241,12 +243,22 @@ cleanup(<<"delete">>, State) ->
 import_rates_into_ratedeck(Ratedeck, {0, []}) ->
     RatedeckDb = kzd_ratedeck:format_ratedeck_db(Ratedeck),
     kz_datamgr:enable_change_notice(),
+    lager:debug("importing into ratedeck '~s' complete", [RatedeckDb]),
     kzs_publish:publish_db(RatedeckDb, <<"edited">>);
-import_rates_into_ratedeck(Ratedeck, {_, Rates}) ->
+import_rates_into_ratedeck(Ratedeck, {_C, Rates}) ->
+    RatedeckDb = kzd_ratedeck:format_ratedeck_db(Ratedeck),
+    lager:debug("importing ~p rate~s into ratedeck '~s'"
+               ,[_C, maybe_plural(_C), RatedeckDb]
+               ),
+
     kz_datamgr:suppress_change_notice(),
-    save_rates(kzd_ratedeck:format_ratedeck_db(Ratedeck), Rates),
+    save_rates(RatedeckDb, Rates),
     import_rates_into_ratedeck(Ratedeck, {0, []}).
 
+-spec maybe_plural(pos_integer()) -> [] | [$s].
```

Comment:
Specs can't get char syntax for some reason. Plus: `[] | [something()]` is exactly `[something()]`.
How about just `string()`?